ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionLayerManagerImpl.java

```diff
@@ -223,6 +226,24 @@ public SafeFuture<PowBlock> eth1GetPowChainHead() {
         "calling engineForkChoiceUpdated(forkChoiceState={}, payloadAttributes={})",
         forkChoiceState,
         payloadBuildingAttributes);
+    if (spec.atSlot(forkChoiceState.getHeadBlockSlot())
```

Comment:
Minor difference but I'd probably switch based on whether capella is enabled at all, even if it hasn't activated yet. We should be able to use V2 for Bellatrix fine and it would be good to test that but without changing the current Mainnet/testnet behaviour since ELs won't yet support V2.